src/index.js

```diff
@@ -0,0 +1,142 @@
+let currentTemperature = 0;
+
+const tempValue = document.getElementById('tempValue');
+
+const temperatureDisplay = () => {
+    tempValue.textContent = currentTemperature;
+    tempColor();
+};
+
+const addDegree = () => {
+    currentTemperature += 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const subtractDegree = () => {
+    currentTemperature -= 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const tempColor = () => {
+    const BackgroundColor = document.getElementById('currentTempButton');
+
+    const elementFontColor = document.getElementById('tempValue');
+    let currentColor;
+    
+    const elementLandscape = document.getElementById('landscape');
+    let currentLandscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+
+    if (currentTemperature <= 49) {
+        currentColor = '#30849D';
+    } else if (currentTemperature >= 50 && currentTemperature <= 59 ) {
+        currentColor = '#50A63F';
+    } else if (currentTemperature >= 60 && currentTemperature <= 69 ) {
+        currentColor = '#FEE086';
+        currentLandscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (currentTemperature >= 70 && currentTemperature <= 79 ) {
+        currentColor = '#F8B5A2';
+        currentLandscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (currentTemperature >= 80) {
+        currentColor = '#E06767';
+        currentLandscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    }
```

Comment:
Instead of using if/else if/else which can be brittle and bug-prone, consider using a list to store the landscape emojis. Then you can write logic that compares the first element in each inner list against `temperature ` and grab the color or emojis. 

```js
temperatureColorsAndLandscapes = [
    [80, "red", "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"],
    [70, "orange", "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"],
    [60, "yellow", "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"],
    [50, "green", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
    [40, "teal", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
]
```